pallets/parachain-staking/src/lib.rs

```diff
@@ -2527,61 +2067,11 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Check for remaining delegations of the delegator which has been
-		/// removed from the given collator.
-		///
-		/// Returns the removed delegator's address and
-		/// * Either the updated delegator state if other delegations are still
-		///   remaining
-		/// * Or `None`, signalling the delegator state should be cleared once
-		///   the transaction cannot fail anymore.
-		fn prep_kick_delegator(
```

Comment:
Not required anymore because the replaced delegator cannot have further delegations.